staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go

```diff
@@ -158,8 +166,12 @@ func newWatchCache(
 		listResourceVersion: 0,
 		clock:               clock.RealClock{},
 		versioner:           versioner,
+		newFunc:             newFunc,
 	}
 	wc.cond = sync.NewCond(wc.RLocker())
+	if utilfeature.DefaultFeatureGate.Enabled(features.WatchBookmark) {
+		go wait.Until(wc.sendBookmark, time.Second, wait.NeverStop)
```

Comment:
Should the sendBookmark sends be jittered via `JitterUntil`? If they don't need to happen exactly every second, this can help level out any aggregate load.

Reply:
Makes a lot of sense to me - I would like to avoid making the jitter large, but something like 25% or sth like that makes a lot of sense.

Reply:
done